DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -437,7 +441,15 @@ public void DeactivateOrDeleteAdmin(int adminId)
             }
             else
             {
-                userDataService.DeleteAdminUser(adminId);
+                try
+                {
+                    userDataService.DeleteAdminUser(adminId);
+                }
+                catch (Exception ex)
+                {
+                    logger.LogWarning(ex, $"Error attempting to delete admin {adminId} with no sessions, deactivating them instead.");

```

Comment:
Do we want to include the exception message in log message on the off chance something else has occurred?

Reply:
With this method, the message will be logged to the Message and MessageTempate columns, and the exception to the Exception column